controllers/managedocs_controller_test.go

```diff
@@ -61,6 +63,23 @@ var _ = Describe("ManagedOCS controller", func() {
 				Expect(k8sClient.Create(ctx, managedOCS)).Should(Succeed())
 				Expect(k8sClient.Get(ctx, getResourceKey(managedOCS), managedOCS)).Should(Succeed())
 
+				// No storage cluster should be created if the addon param secret does not exist
+				Expect(k8sClient.List(ctx, scList, client.InNamespace(TestNamespace))).Should(Succeed())
+				Expect(scList.Items).Should(HaveLen(0))
+
+				data := make(map[string][]byte)
+				data["size"] = []byte("1Ti")
+
+				secret := &corev1.Secret{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      ConfigSecretName,
+						Namespace: TestNamespace,
+					},
+					Data: data,
+				}
+				Expect(k8sClient.Create(ctx, secret)).Should(Succeed())
+				Expect(k8sClient.Get(ctx, getResourceKey(secret), secret)).Should(Succeed())
+
```

Comment:
This should be a separate test case that runs even before the  first one

Reply:
done.